sorting/go/bubble-sort.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"math"
+	"os"
+	"strconv"
+	"strings"
+)
+
+//Swaps two values of a slice
+func Swap(sliceInt []int, indexValue int) {
+	//Treats edge case
+	if indexValue < len(sliceInt)-1 {
+		tempValue := sliceInt[indexValue]
+		sliceInt[indexValue] = sliceInt[indexValue+1]
+		sliceInt[indexValue+1] = tempValue
+	}
+}
+
+//Recursive implementation of Bubble sort
+func bubbleSort(sliceInt []int) {
+	for k := 0; k < len(sliceInt)-1; k++ {
+		if sliceInt[k] > sliceInt[k+1] {
+			Swap(sliceInt, k)
+			bubbleSort(sliceInt)
+		}
+	}
+}
+
+func check(e error) {
+	if e != nil {
+		fmt.Println("error: ", e)
+	}
+}
+
+func userInput() []int {
```

Comment:
Also the `userInput()` part is making the code too complex, the aim of this code should be demonstrating how bubble sort works so I suggest you should completely remove this part and take a hardcoded slice and perform bubble sort on that.